src/presentational/Layout.jsx

```diff
@@ -75,7 +75,8 @@ class Layout extends Component {
         </main>
 
         <footer className='site-footer'>
-          &nbsp;
+          <p className='pull-left'>Zooniverse.org - The universe is too big to explore without you</p>
+          <p className='pull-right'>2016</p>
```

Comment:
I like this footer much better! But we should go whole hog and change that Zooniverse.org label into a full `<a>` link